SConstruct

```diff
@@ -185,12 +185,25 @@ if platform == "darwin":
 					break
 elif platform == "win32":
 	if 'msvc' in env['TOOLS']:
-		vcpkg_prefix = (os.environ['HOME'] if 'HOME' in os.environ else 'C:') + f'/vcpkg/installed/x{env["bits"]}-windows'
+		vcpkg_prefix = path.join((os.environ['HOME'] if 'HOME' in os.environ else 'C:\\'), 'vcpkg')
+		vcpkg_installed = path.join(vcpkg_prefix, 'installed', f'x{env["bits"]}-windows')
+		vcpkg_other_packages = Glob(path.join(vcpkg_prefix, 'packages', f'**x{env["bits"]}-windows'))
+		vcpkg_other_includes = [path.join(d.get_abspath(), 'include') for d in vcpkg_other_packages]
+		vcpkg_other_includes = list(filter(path.exists, vcpkg_other_includes))
+		vcpkg_other_libs = [path.join(d.get_abspath(), 'lib') for d in vcpkg_other_packages]
+		vcpkg_other_libs = list(filter(path.exists, vcpkg_other_libs))
+		vcpkg_other_bins = [path.join(d.get_abspath(), 'bin') for d in vcpkg_other_packages]
+		vcpkg_other_bins = list(filter(path.exists, vcpkg_other_bins))
+		project_includes = []
+		for (root, dirs, files) in os.walk('src'):
+			project_includes.append(path.join(os.getcwd(), root))
+
+		include_paths=[path.join(vcpkg_installed, 'include')] + vcpkg_other_includes + project_includes
 		env.Append(
-			LINKFLAGS=['/SUBSYSTEM:WINDOWS','/ENTRY:mainCRTStartup','/MACHINE:X86'],
+			LINKFLAGS=['/SUBSYSTEM:WINDOWS','/ENTRY:mainCRTStartup',f'/MACHINE:X{env["bits"]}'],
```

Comment:
Does `/MACHINE:X32` actually work though? Because that's what this is going to pass in if you try to do a win32 build.